### PR TITLE
[candi] refactor bootstrap scripts to fit 16kb limit of cloud-data

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

Refactor bootstrap scripts to fit 16kb limit of cloud-data.
## Why do we need it, and what problem does it solve?

In AWS cloud provider size of bootstrap scripts is limited to 16kb. We need to precisely write the bash bootstrap code to avoid exceeding the limit.
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: refactor bootstrap scripts to fit 16kb limit of cloud-data
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/aws'...

**Comment by RomanenkoDenys** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 4894509507 4894509507-1-con-1-23 /sys/deckhouse-oss/install:pr4580 centos@35.158.46.45

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/aws'...

**Review by nabokihms — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by RomanenkoDenys** [comment]
/e2e/abort vsphere;Standard;containerd;1.23 4928680566 4928680566-1-con-1-23 /sys/deckhouse-oss/install:pr4580 ubuntu@10.220.203.26

**Comment by RomanenkoDenys** [comment]
/e2e/abort openstack;Standard;containerd;1.23 4928677944 4928677944-1-con-1-23 /sys/deckhouse-oss/install:pr4580 debian@188.124.39.13

**Comment by RomanenkoDenys** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.23 4928676401 4928676401-2-con-1-23 /sys/deckhouse-oss/install:pr4580 user@34.159.41.220

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by RomanenkoDenys** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.23 4928676401 4928676401-1-con-1-23 /sys/deckhouse-oss/install:pr4580 user@34.159.60.160

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/gcp'...

**Comment by z9r5** [comment]
@RomanenkoDenys Please fill in the "Why do we need it in the patch release" section.

**Review by yalosev — APPROVED** [review]

File: modules/040-node-manager/templates/node-group/_bootstrap_script.tpl [hunk 1]
```diff
@@ -149,7 +149,7 @@ fi
   {{- /*
 # IMPORTANT !!! Centos/Redhat put jq in /usr/local/bin but it is not in PATH.
   */}}
-export PATH="/usr/local/bin:$PATH"
+export PATH="/opt/deckhouse/bin:$PATH"
```
Comment (konstantin-axenov) [inline modules/040-node-manager/templates/node-group/_bootstrap_script.tpl:152]: Can it be backported in 1.45?

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**